cypress/e2e/BlogSeriesBlogPosts.feature

```diff
@@ -50,7 +50,6 @@ Feature: As an user, I want to see different components of the Blog Series and B
             | 2019  | false      |
             | 2018  | false      |
             | 2017  | false      |
-            | 2016  | false      |
```

Comment:
Along with the removal of years which are no longer available, should this also be expanded to include 2020 through 2023?